model/qti/ParserFactory.php

```diff
@@ -353,6 +364,20 @@ protected function parseContainerItemBody(DOMElement $data, ContainerItemBody $c
             }
         }
 
+        // parse for tables, but only the ones containing interactions
```

Comment:
That's because tables may contains interactions : the current block will call parseContainerInteractive so the interactions that the tables may contain are properly parsed.